html/modules/custom/reliefweb_import/src/Plugin/ReliefWebImporter/InoreaderImporter.php

```diff
@@ -387,6 +388,7 @@ protected function processDocuments(array $documents, string $provider_uuid, Con
         'status' => 'pending',
         'message' => '',
         'attempts' => 0,
+        'source' => trim(substr($document['origin']['title'] ?? '', 0, strpos($document['origin']['title'] ?? '', '[source:'))),
```

Comment:
```suggestion
        'source' => trim(substr($document['origin']['title'] ?? '', 0, strpos($document['origin']['title'] ?? '', '[source:') ?: NULL)),
```

Use the full title if `[source:` could not be found.